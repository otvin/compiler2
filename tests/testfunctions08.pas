program testfunctions08(output);
{tests real params and returns in functions}


function realfunc2(r:Real):Real;
begin
    realfunc2 := r * 2.0;
end;

begin
    writeln(realfunc2(13.0));
end.
