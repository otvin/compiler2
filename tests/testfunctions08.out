26.000000000000
